### ISSUE TITLE
[BUG] Can't start a game after Patch 0.207.20

### ISSUE THREAD

**Issue by fitzfuchs** [valheimPlus/ValheimPlus#679]
- **OS:** Windows 11
- **Valheim Game Version: 0.207.20
- **V+ Mod Version: 0.9.9.7

**Describe the bug**

Error in Unity Log

**To Reproduce**

Start the game.

**Error**

[Error  : Unity Log] ArgumentException: Undefined target method for patch method static System.Collections.Generic.IEnumerable<HarmonyLib.CodeInstruction> ValheimPlus.GameClasses.Chat_AddString_Transpiler::Transpiler(System.Collections.Generic.IEnumerable<HarmonyLib.CodeInstruction> instructions)

**Comment by MarcusSjolin** [comment]
Have same issue.

So this makes V+ crash when starting the game, however the game starts properly and presents a white big box in the main screen and V+ is not present when logging into a game session.

**Comment by nxPublic** [comment]
On it.

**Comment by qkdxorjs1002** [comment]
```
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type Chat and name AddString and parameters 
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static System.Collections.Generic.IEnumerable<HarmonyLib.CodeInstruction> ValheimPlus.GameClasses.Chat_AddString_Transpiler::Transpiler(System.Collections.Generic.IEnumerable<HarmonyLib.CodeInstruction> instructions)
Stack trace:
HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
Rethrow as HarmonyException: Patching exception in method null
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
ValheimPlus.ValheimPlusPlugin.Awake () (at <05d4279945b1490b848a7812f51331f5>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Application:.cctor()
```
```
[Error  : Unity Log] MissingMethodException: void Chat.AddString(string,string,Talker/Type)
Stack trace:
(wrapper dynamic-method) Chat.DMD<Chat::OnNewChatMessage>(Chat,UnityEngine.GameObject,long,UnityEngine.Vector3,Talker/Type,string,string)
Chat.RPC_ChatMessage (System.Int64 sender, UnityEngine.Vector3 position, System.Int32 type, System.String name, System.String text) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Delegate.DynamicInvokeImpl (System.Object[] args) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Delegate.DynamicInvoke (System.Object[] args) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
RoutedMethod`4[T,U,V,B].Invoke (System.Int64 rpc, ZPackage pkg) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
ZRoutedRpc.HandleRoutedRPC (ZRoutedRpc+RoutedRPCData data) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
ZRoutedRpc.InvokeRoutedRPC (System.Int64 targetPeerID, ZDOID targetZDO, System.String methodName, System.Object[] parameters) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
ZRoutedRpc.InvokeRoutedRPC (System.Int64 targetPeerID, System.String methodName, System.Object[] parameters) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
Chat.SendText (Talker+Type type, System.String text) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
Game.UpdateRespawn (System.Single dt) (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
Game.FixedUpdate () (at <dcb0de0155834d459f18d7ac6ea5244b>:0)
```

**Comment by Kranglefanten** [comment]
My game starts fine, but i do have a white box on the screen and this message shows up in the BepInEx. And only parts of V+ is working. 

![Screenshot_776](https://user-images.githubusercontent.com/100699500/156184932-7c4d139e-efe9-4d24-a26c-b8d504c721ab.png)

**Comment by nxPublic** [comment]
Should be addressed with 0.9.9.8.

Please report any additional issues or errors in a new thread / bug report.
